dataset_conversion/convert_bids_to_nnUnetv2.py

```diff
@@ -50,11 +52,71 @@ def get_parser():
     parser.add_argument('--split', nargs='+', type=float, default=[0.8, 0.2],
                         help='Ratios of training (includes validation) and test splits lying between 0-1. '
                              'Example: --split 0.8 0.2')
-
     return parser
 
 
+def convert_subject(root, subject,channel, contrast, label_suffix, path_out_images, path_out_labels, counter, list_images,
+                    list_labels, is_ses, session=None):
+    """Function to get image from original BIDS dataset modify if needed and place
+        it with a compatible name in nnUNet dataset.
+
+    Args:
+        root (str): Path to BIDS dataset directory.
+        subject (str): Subject name.
+        contrast (str): Type of contrast.
+        label_suffix (str): suffix of the label in derivatives.
+        path_out_images (str): path to the images directory in the new dataset (test or train).
+        path_out_labels (str): path to the labels directory in the new dataset (test or train).
+        counter (int): Number of subject already treated.
```

Comment:
this is not clear - what is meant by "treating" a subject? 

maybe something like "counter for iterating through the number of subjects"